src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.stories.js

```diff
@@ -2169,6 +2169,31 @@ const exampleInputChoiceSet = {
   ],
 };
 
+const exampleInputToggle = {
+  $schema: 'http://adaptivecards.io/schemas/adaptive-card.json',
+  type: 'AdaptiveCard',
+  version: '1.0',
+  body: [
+    {
+      type: 'Input.Toggle',
+      id: 'acceptTerms',
+      title: 'I accept the terms and agreements',
+      value: 'true',
```

Comment:
let's demonstrate the full functionality of these fields:
``js
value: 'rejected',
valueOn: 'accepted',
valueOff: 'rejected',